homeassistant/components/nest/camera_sdm.py

```diff
@@ -90,11 +96,24 @@ async def stream_source(self):
         if CameraLiveStreamTrait.NAME not in self._device.traits:
             return None
         trait = self._device.traits[CameraLiveStreamTrait.NAME]
-        rtsp_stream = await trait.generate_rtsp_stream()
+        now = datetime.datetime.now(tz=datetime.timezone.utc)
```

Comment:
We have the utcnow util.
https://github.com/home-assistant/core/blob/2a795c039742c31a72323e382a98e89e0d1fbfa9/homeassistant/util/dt.py#L54